manifests/poc-data-cluster/prod/helium/iot-rewards-share-delta-lake-sink.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: iot-rewards-share-delta-lake-sink
+  namespace: helium
+spec:
+  concurrencyPolicy: Forbid
+  schedule: "10 */4 * * *"
+  jobTemplate:
+    spec:
+      backoffLimit: 10
+      template:
+        spec:
+          serviceAccountName: s3-data-lake-bucket-access
+          tolerations: # Schedule executor pods on spot instance group
+            - key: dedicated
+              operator: Equal
+              value: spark
+              effect: NoSchedule
+          nodeSelector:
+            nodegroup-type: spot
+          containers:
+          - name: iot-rewards-delta-lake-sink
+            image: public.ecr.aws/k0m1p4t7/protobuf-delta-lake-sink:0.0.9
+            imagePullPolicy: IfNotPresent
+            resources:
+              requests:
+                cpu: 1000m
+                memory: 6900Mi
+              limits:
+                memory: 6900Mi
+            env: 
+              - name: AWS_S3_ALLOW_UNSAFE_RENAME
+                value: "true"
+            args:
+              - --source-bucket
+              - foundation-poc-data-requester-pays
+              - --source-region
+              - us-west-2
+              - --file-prefix
+              - foundation-iot-verified-rewards/iot_reward_share
+              - --source-proto-name
+              - "iot_reward_share"
+              - --source-proto-base-url
+              - https://raw.githubusercontent.com/helium/proto/master/src
+              - --source-protos
+              - data_rate.proto
+              - --source-protos
+              - region.proto              
+              - --source-protos
+              - service/poc_lora.proto
```

Comment:
I don't think this is the right proto. see the helium-data script

Reply:
What is wrong here?